101-cosmosdb-create-arm-gremlin/metadata.json

```diff
@@ -0,0 +1,7 @@
+{ 
+   "itemDisplayName": "Create a Cosmos Database Account using the Gremlin API", 
+   "description": "This template creates a Cosmos Database Account using the Gremlin API",
```

Comment:
Change to "This template creates an Azure Cosmos DB Gremlin (Graph) API account."